packages/typescript-eslint/src/config-helper.ts

```diff
@@ -85,11 +85,34 @@ export interface ConfigWithExtends extends TSESLint.FlatConfig.Config {
 export function config(
   ...configs: ConfigWithExtends[]
 ): TSESLint.FlatConfig.ConfigArray {
-  return configs.flatMap(configWithExtends => {
+  return configs.flatMap((configWithExtends, configIndex) => {
     const { extends: extendsArr, ...config } = configWithExtends;
     if (extendsArr == null || extendsArr.length === 0) {
       return config;
     }
+    const undefinedExtensions = extendsArr.reduce<number[]>(
+      (acc, extension, extensionIndex) => {
+        const maybeExtension = extension as
+          | TSESLint.FlatConfig.Config
+          | undefined;
+        if (maybeExtension == null) {
+          acc.push(extensionIndex);
+        }
+        return acc;
+      },
+      [],
+    );
+    if (undefinedExtensions.length) {
+      const configName =
+        configWithExtends.name != null
+          ? `, named "${configWithExtends.name},"`
```

Comment:
```suggestion
          ? `, named "${configWithExtends.name}",`
```

Reply:
afaik, in AE commas always go before the closing quotation. but i have no strong opinion here.

Reply:
As a native AE speaker, I was taught this abominable style in the elementary school classroom too 🤣 I have since been rehabilitated 😆

Reply:
feel free to commit your own suggestion, or not, as you merge this PR

Reply:
Oh yeah the putting of punctuation _inside_ the quotes is something I've always hated too. It's weirdly ambiguous. Strong +1 to kirk's suggested change. 😄